.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
```

Comment:
What is this file for?

Reply:
A unified CI test instead of Appveyor (and Travis) that fails building Windows.
https://ci.appveyor.com/project/andreasnoack/mkl-jl/builds/30808306

Reply:
I think we can just use Travis (for all 3 OS).

Reply:
That is possible, but Github actions is IMO faster.

Reply:
If they are faster we should probably use it. If I merge this will it just start to become active? There are no new notifications from this PR.

Reply:
They are based on my tests. 

I think they should get active once merged. Probably it is because I am making a PR to your branch instead of Master.

Reply:
I don't think they will pass though because the default test script is not suitable to test this packge (which is why I have the custom stuff in the .travis in the PR). Can we just first put in CompatHelper + TagBot and then we can deal with changing CI later?

Reply:
Ahh OK. I am going to delete the file for now.